src/quickgui/plugin/qgsquickmapcanvas.qml

```diff
@@ -16,7 +16,7 @@
 import QtQuick 2.3
 import QtQuick.Controls 2.2
 import QtQml 2.2
-import QgisQuick 0.1 as QgsQuick
+import QgsQuick 0.1 as QgsQuick
```

Comment:
`import QgsQuick 0.1` ?

Reply:
I like to have namespace before the class in the code, mainly for better googling: for example

`QgsQuick.MapSettings` ---> I immediately know it comes from `qgsquickmapsettings.h`

but 

`MapSettings`,  it is not that clear where is it defined.

Maybe it makes more sense with some elements that could be defined in various libraries, for example scalebar or so.

But I really do not have a strong opinion on this one